src/RoveComm/RoveCommUDP.cpp

```diff
@@ -0,0 +1,499 @@
+/******************************************************************************
+ * @brief The RoveCommUDP class is used to send and receive data over a UDP
+ *        connection.
+ *
+ * @file RoveCommUDP.cpp
+ * @author Eli Byrd (edbgkk@mst.edu)
+ * @date 2024-02-07
+ *
+ * @copyright Copyright Mars Rover Design Team 2024 - All Rights Reserved
+ ******************************************************************************/
+
+#include "RoveCommUDP.h"
+#include "RoveCommPacket.h"
+#include <arpa/inet.h>
```

Comment:
Put implicit include in 
/// \cond
#include <array>

/// \endcond
to avoid them being included in doxygen graph.